code/common/datadog.q

```diff
@@ -4,19 +4,29 @@
 \d .dg
 
 //default to disabled - .lg.ext will not be overwritten
-enabled:@[value;`enabled;0b] 
+enabled:@[value;`enabled;0b]
+
+//default to disabled - datadog agent used
+webreq:@[value;`webreq;0b] 
 
 //define dogstatsd_port
 dogstatsd_port:@[value;`dogstatsd_port;getenv[`DOGSTATSD_PORT]]
 
+//define dogstatsd_apikey
+dogstatsd_apikey:@[value;`dogstatsd_apikey;getenv[`DOGSTATSD_APIKEY]]
+
+//define dogstatsd_url
+dogstatsd_url:@[value;`dogstatsd__url;getenv[`DOGSTATSD_URL]]
```

Comment:
Typo, extra underscore:
```suggestion
dogstatsd_url:@[value;`dogstatsd_url;getenv[`DOGSTATSD_URL]]
```
Is this URL ever likely to change much? If not then maybe it doesn't need to be an environment variable and something that can be overwritten easily.

Reply:
Would you suggest just defining it within the file? Or where would be the best place to put it?

Reply:
Probably just here and drop from `setenv.sh`:
```suggestion
dogstatsd_url:":https://api.datadoghq.com/api/v1/";
```